packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -52,24 +58,32 @@ export const ContextualMenu = compose<ContextualMenuType>({
 
     const slotProps = mergeSettings<ContextualMenuSlotProps>(styleProps, {
       root: {
-        ...rest
+        ...rest,
+        setInitialFocus: shouldFocusOnMount
+      },
+      container: {
+        accessible: shouldFocusOnContainer,
+        ... { acceptsKeyboardFocus: shouldFocusOnContainer },
```

Comment:
What effect does `... { acceptsKeyboardFocus: shouldFocusOnContainer }` have over `acceptsKeyboardFocus: shouldFocusOnContainer` ?

Reply:
IIRC the spread operator lets us "sneak" the property through past type checking against ViewProps, if ViewProps doesn't have the property.  @lenahong maybe a comment for what issue we're addressing by adding it?

Reply:
I guess type checking does not occur for this code bit, because I tried changing to just `acceptsKeyboardFocus: shouldFocusOnContainer` and it seems to build and work fine.

Reply:
I see, thanks for the explanation!